composer.json

```diff
@@ -55,7 +55,7 @@
     "lint" : "phpcs",
     "fix" : "phpcbf",
     "stan": "phpstan analyse --ansi --memory-limit 2048M",
-    "phpstan-baseline": "phpstan analyse --ansi --generate-baseline",
+    "baseline": "phpstan analyse --ansi --generate-baseline",
```

Comment:
:pray: Shortcuts should be short. Totally agree with the change

Reply:
😢

Reply:
@simPod I take it you have an objection..?

Reply:
Yup, was discussed previously though. Don't fancy those shortcuts as they're not descriptive, are ambiguous and save nothing anyway as every decent shell fills it for you anyway.

Reply:
Ah, I see. I use gitbash, and have never noticed any auto-completion. Am I using a bad shell?

Oh lord, I've stumbled into a holy war. Speaking as someone who is a heavy user of the baseline functionality, I grew tired of having to check composer.json to see if it was "baseline" or "phpstan-baseline" or "stan-baseline" or "phpstanbaseline", etc, and this seemed to lift that block. As for the former being more descriptive, I figure neither "baseline" nor "phpstan-baseline" are going to make sense to anyone who is not familiar with the concept, and if you are, then both are equally informative.

If you're okay with this for now, I'll leave it as-is...

Reply:
that's why I only 😢 and passed by. 

It's phpstan baseline tho, not some baseline. If we started to use baseline from another tool, the conflict is already there. Same is with lint, fix, stan etc. In my opinion it's broken already so doesn't really matter.

yes I guess :P

Reply:
>  If we started to use baseline from another tool, the conflict is already there

Fair enough, but the way I look at it phpstan baseline is our "default" baseline. If another one gets added in the future, we can add it with a more specific key. That's the way I'm generally used to doing it, but in any case we can certainly revisit all this if new tools do appear and compete for the namespace.

Reply:
I fancy open closed principle also in cases like this. That's why I'm against changes like this. BC breaks happen then and DX suffer.

Reply:
> and DX suffe

Speaking as a D, the X was already suffering. But your point is taken and understood.

Reply:
Pragmatism vs purity @shmax ? 🤔 Sorry, couldn't resist temptation 😄

Reply:
Shots fired! I love you guys, but it seems like in a lot of these discussions we fall into the false dichotomy trap. We are not limited to choosing between "baseline" and "phpstan-baseline". I'm not trying to impress anybody, here, but at work I'm used to seeing script sections that are far, far more complex than this (with 20 or more commands). They usually wind up taking on a recursive form, where broadly-named scripts invoke the more specific ones. You could have your `phpstan-baseline` script, and have the "default" `baseline` script invoke it. Any other baseline tools that appear likewise get specific names, and they may or may not be invoked by the default one. 

I rather like the simplicity of what we have, but we don't have to sweat blood over this (I could always just alias it in my shell, anyway). If ya'll would rather sort this out in some other PR I don't object.